packages/plugin-atoms/src/components/Image.tsx

```diff
@@ -0,0 +1,43 @@
+import { useComponentUtils } from '@ima/react-page-renderer';
+import { ComponentPropsWithoutRef, memo } from 'react';
+import type { ValueOf } from 'type-fest';
+
+import { DECODING, LAYOUT, LOADING } from './constants';
+
+export const Image = memo(function ImageComponent({
+  src,
+  layout,
+  className,
+  loading = LOADING.LAZY,
+  decoding = DECODING.ASYNC,
```

Comment:
to je vlastně to co píšu dole, za mě je to zbytečný overhead a `'async' | 'lazy'`, nikde se na tu konstantu stejně neodkazuješ tak to je zbytečné.

Reply:
decoding is set to 'async'